### PR TITLE
unify cli and web tailwind config colors with figma colors

### PR DESCRIPTION
**Pull request by monicakogler**
Screen Shot 2022-10-14 at 10 36 39 AM
Screen Shot 2022-10-14 at 10 37 02 AM

### CODE REVIEW

**Review by psugihara — CHANGES_REQUESTED** [review]
I think this is caused by the blue key, not sure:

Screen Shot 2022-10-14 at 11 08 27 AM

**Comment by monicakogler** [comment]
woops thanks. going with the alt syntax rather than replacing all instances of `blue`

**Review by psugihara — APPROVED** [review]
looks good!

File: packages/web/tailwind.config.js [hunk 1]
```diff
@@ -7,20 +7,28 @@ module.exports = {
   theme: {
```
Comment (alexfarrill) [inline packages/web/tailwind.config.js:7]: @psugihara we should be able to symlink this, no?

Comment (psugihara) [reply packages/web/tailwind.config.js:7]: sure i'll try

**Pull request merged**